### PR TITLE
added test/tokencapgc.js {garbage test}

### PR DESCRIPTION
**Pull request by dev-matan-tsuberi**

### CODE REVIEW

File: test/tokencapgc.js [hunk 1]
```diff
@@ -45,4 +45,18 @@ contract('TokenCapGC', function (accounts)  {
...
   });
+
+  it('garbage transaction test', async () => {
+    const tokenCapGC = await TokenCapGC.new();
+    const addr = tokenCapGC.address;
+
+    // transaction should fail.
+    try{
+      await web3.eth.call({to: addr, data: 0x231ef231ef231ef});
+      assert.fail(true,false);
```
Comment (orenyodfat) [inline test/tokencapgc.js]: It worth throwing some meaningful message here..
Take a look [here](https://github.com/daostack/daostack/blob/master/test/schemeregistrar.js#L77) for reference

Comment (dev-matan-tsuberi) [reply test/tokencapgc.js]: Thanks, will do.

**Comment by orenyodfat** [comment]
Why this test is needed ? It is general solidity behavior..calling a contract without a fallback func..
Most of our contacts does not have fallbacks.

**Comment by dev-matan-tsuberi** [comment]
I agree, it's a trivial test. 
I'm new here so I'm just making sure I have everything set up correctly.

**Comment by orenyodfat** [comment]
👍

**Pull request closed without merging**